### ISSUE TITLE
Dynamic updates to "sample_weight" in fit() for importance sampling (IS) weights or similar use cases.

### ISSUE THREAD

**Issue by WorksWellWithOthers** [keras-team/keras#13533]
Feature request if it doesn't already exist. I was hoping there was a feature that wouldn't require me to make a custom loss function to multiply a different weight with the mean_squared_error each time fit() was called. 

Is "sample_weight" dynamic / updated each time fit() is called?

Does the "sample_weight" option in fit(),
 `model.fit(states, next_action, sample_weight=is_weights, epochs=1, verbose=0)`
alleviate the need to do the solution suggested here?

[Keras Loss Function with Additional Dynamic Parameter](https://stackoverflow.com/questions/50124158/keras-loss-function-with-additional-dynamic-parameter/50127646])

**Comment by Arktius** [comment]
Have you already found a solution? I'm looking for a way to weight each training sample as well as for each validation sample.
Moreover, the feature that I need has to consider weights not only during training but also during inference.
